### PR TITLE
fix `karmor uninstall`

### PR DESCRIPTION
**Pull request by Ankurk99**
**Description**
- Uninstall KubeArmor Daemonset
- Uninstall all the resources based on label (kubearmor-app)
- `karmor uninstall` resources uninstall it from all the ns and
- `karmor uninstall -n <ns>` uninstalls the resources from that particular ns
- fixed the rotating status cursor issue on terminal resize

### CODE REVIEW

File: install/install.go [hunk 1]
```diff
@@ -756,35 +793,59 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	}
 
 	fmt.Print("❌   Cluster Role ...\n")
-	if err := c.K8sClientset.RbacV1().ClusterRoles().Delete(context.Background(), deployments.KubeArmorClusterRoleName, metav1.DeleteOptions{}); err != nil {
-		if !strings.Contains(err.Error(), "not found") {
-			return err
+	clusterRoleList, err := c.K8sClientset.RbacV1().ClusterRoles().List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app"})
```
Comment (daemon1024) [inline install/install.go:796]: Is this backwards compatible?
We don't have the label selector in the current Cluster Role right?

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: cmd/uninstall.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ var uninstallCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(uninstallCmd)
 
-	uninstallCmd.Flags().StringVarP(&uninstallOptions.Namespace, "namespace", "n", "kubearmor", "Namespace for resources")
+	uninstallCmd.Flags().StringVarP(&uninstallOptions.Namespace, "namespace", "n", "", "Namespace for resources")
```
Comment (DelusionalOptimist) [inline cmd/uninstall.go:27]: I think we should add to the help message what would the default behavior be when no namespace is specified? :thinking:

File: install/install.go [hunk 3]
```diff
@@ -633,181 +635,348 @@ func removeAnnotations(c *k8s.Client) {
...
 }
 
+func contains(slice []string, item string) bool {
+	for _, s := range slice {
+		if s == item {
+			return true
+		}
+	}
+	return false
+}
```
Comment (daemon1024) [inline install/install.go:645]: Use https://pkg.go.dev/slices#Contains

**Review by daemon1024 — CHANGES_REQUESTED** [review]
small nit, tried it out locally. Works as intended. LGTM 🙌🏽

File: install/install.go [hunk 4]
```diff
@@ -12,6 +12,7 @@ import (
 	"fmt"
 	"os"
 	"path"
+	"slices"
```
Comment (DelusionalOptimist) [inline install/install.go:15]: I think this package is still in experimental before Go 1.21.
Either need to update go.mod and CI workflows OR use `"golang.org/x/exp/slices"`

**Review by DelusionalOptimist — APPROVED** [review]
LGTM! Thanks @Ankurk99. :rocket:

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**